sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestRecord.java

```diff
@@ -35,12 +35,15 @@
 
     private static final Logger logger = LoggerFactory.getLogger(RntbdRequestRecord.class);
```

Comment:
Same as above. I want to compare what's in my branch to what's in this PR.

Reply:
Also: do we need the "rawtypes" suppression after adding the template types?

Reply:
right, they're not required anymore, removing them.